lte/gateway/c/core/oai/test/mme_app_task/test_mme_procedures.cpp

```diff
@@ -116,10 +116,10 @@ class MmeAppProcedureTest : public ::testing::Test {
   virtual void TearDown() {
     bdestroy_wrapper(&nas_msg);
     send_terminate_message_fatal(&task_zmq_ctx_main);
+    // Sleep to ensure that messages are received and contexts are released
+    std::this_thread::sleep_for(std::chrono::milliseconds(500));
```

Comment:
We can define macro for 500

Reply:
Since it is only used here now, I did not do that.